tests/integration/test_mysql_protocol/prepared_statements_test.sql

```diff
@@ -0,0 +1,127 @@
+CREATE TABLE ps_simple_data_types
```

Comment:
Let's add either `DROP TABLE ps_simple_data_types` before l. 1 (to remove remnants) OR replace `CREATE TABLE [...]` by `CREATE OR REPLACE TABLE [...]`